### PR TITLE
used existing JSON library to greatly simplify things

### PR DESCRIPTION
**Pull request by gmartin7**
Also removed an unused method

### CODE REVIEW

File: src/BloomExe/Edit/EditingModel.cs [hunk 1]
```diff
@@ -1196,26 +1188,28 @@ private string GetPathToCurrentTemplateHtml
...
-				var jsonString = JSON_START + jsonBookFolder + JSON_DIVIDER + jsonBook + JSON_DIVIDER + jsonLastPage + JSON_END;
-				return jsonString;
+				var addPageSettings = new Dictionary<string, object>();
+				var folderPath = MassageUrlForJavascript(Path.GetDirectoryName(GetPathToCurrentTemplateHtml));
+				addPageSettings.Add("templateBookFolderUrl", folderPath);
+				var htmlFilePath = MassageUrlForJavascript(GetPathToCurrentTemplateHtml);
+				addPageSettings.Add("templateBookUrl", htmlFilePath);
+				addPageSettings.Add("lastPageAdded", _lastPageAdded);
+				var settingsString = JsonConvert.SerializeObject(new object[] {addPageSettings});
+				return settingsString;
```
Comment (JohnThomson) [inline src/BloomExe/Edit/EditingModel.cs]: I'm not sure what you get when you initialize an object array with a dictionary, but have a look at the technique used in BookMetaData.WebDataJson and see if it wouldn't make things even simpler.

Comment (gmartin7) [inline src/BloomExe/Edit/EditingModel.cs]: As Add Page is currently set up it requires an array of collectionUrl objects. I can't figure out how to get the JsonConvert.SerializeObject method to generate an array other than the way I did it here.

Comment (hatton) [inline src/BloomExe/Edit/EditingModel.cs]: Nice improvement. Note can make it even clearer by using an ExpandoObject():

	dynamic addPageSettings = new System.Dynamic.ExpandoObject();
	addPageSettings.templateBookFolderUrl = MassageUrlForJavascript(Path.GetDirectoryName(GetPathToCurrentTemplateHtml));
	addPageSettings.templateBookUrl = MassageUrlForJavascript(GetPathToCurrentTemplateHtml);
	addPageSettings.lastPageAdded = _lastPageAdded;
	return JsonConvert.SerializeObject(new object[] { addPageSettings });

Comment (gmartin7) [inline src/BloomExe/Edit/EditingModel.cs]: I like it! Never heard of an ExpandoObject before.

**Pull request merged**